src/main/java/com/study/platform/security/WebSecurityConfig.java

```diff
@@ -11,11 +13,16 @@ protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
                 .headers().frameOptions().sameOrigin();
-
         http
-                .authorizeRequests()
-                .antMatchers("/**").permitAll()
-                .and()
-                .cors();
+            .authorizeRequests(a -> a
+                .antMatchers("/", "/error", "/webjars/**").permitAll()
+                .anyRequest().authenticated()
+            )
+            .exceptionHandling(e -> e
```

Comment:
여기도 더 의미잇는 변수가 잇지 않을까요?
```suggestion
            .exceptionHandling(exception -> exception
```